requirements.txt

```diff
@@ -1,3 +1,4 @@
+django-celery-email>=3.0.0,<3.1
```

Comment:
@pandafy this shouldn't be here as it's not a direct dependency of openwisp-notifications. I'm going to remove manually.